project/Http4sPlugin.scala

```diff
@@ -53,7 +53,26 @@ object Http4sPlugin extends AutoPlugin {
     mimaFailOnProblem := http4sMimaVersion.value.isDefined,
     mimaPreviousArtifacts := (http4sMimaVersion.value map {
       organization.value % s"${moduleName.value}_${scalaBinaryVersion.value}" % _
-    }).toSet
+    }).toSet,
+
+    // Override rig's default of the Travis build number being the bugfix number
+    releaseVersion := { ver =>
```

Comment:
The idea here is that we will be releasing whenever we push a tag.  Nothing here ensures that the version number matches the tag.  We could calculate a version number from the tag.

Because we push the tag to trigger a release instead of letting a rig release apply the tag, there is no guarantee that the value in version.sbt on the tag will match the published version. 🤕 

We could alternatively publish off (a) certain branch(es), which is how we use rig at Verizon.  The trick there is what triggers a release build vs. a snapshot build?

Reply:
I've never tried to sneak a tag into a PR before. But is that something we should be worried about with this?

Reply:
Travis [builds tags](https://travis-ci.org/http4s/http4s/builds/276646059) by default, and provides a `TRAVIS_TAG` environment variable for them.